setup.py

```diff
@@ -49,4 +49,4 @@ def requirements_for(version=None):
     description='Library to interface with Project Gutenberg',
     long_description=open('README.rst').read(),
     dependency_links=dependency_links,
-    install_requires=install_requires)
+    install_requires=list(install_requires))
```

Comment:
Should this be `sorted` instead of `list` so that the dependency list is guaranteed to always be the same if the dependencies didn't change?

Reply:
Yeah, I'd rather change this to be sorted or make it so that reading in the files is to a list and then we add lists together. This fix still leaves in a potentially random order which while probably not a problem, deterministic orders are always better in case of weird pip interactions between packages.